### DISCUSSION TITLE
使用yarn create umi创建项目运行不起来

### DISCUSSION THREAD

**Discussion by RaneeX** [umijs/umi#11552] [General]
开发环境：
node： v18.14.2
yarn:     v1.22.19
umi:      v4.0.76

使用yarn create umi命令初始化项目，选了 antd pro的模板 + yarn，自动安装好生成node_modules后卡在了最后一步
![image](https://github.com/umijs/umi/assets/65445560/a16c498c-87aa-4d33-a6f9-30e7c8490f76)

试过了多个node版本（开发常用16.X），也删除重新执行命令过，实在不知道原因出在哪里，特此求助，万分感谢

**Comment by fz6m** [comment]
你们的电脑会自动给代码加密修改代码文件吗？尝试删除 `./src/.umi` 后再启动项目看看，或者尝试用 `pnpm create umi my-app` 来初始化项目看看，另外可以从电脑或者 IDE 的编码格式上找找原因。

**Reply by RaneeX** [reply]
好的谢谢，电脑有加密软件，我试一试新的方式
